examples/olm/catalog-create-subscribe.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 apiVersion: operators.coreos.com/v1
 kind: OperatorSource
 metadata:
-  name: hekumar-operators
+  name: redhat-local-storage-src
```

Comment:
not sure if name of non-existing company is the right one. Origin? OpenShift? okd?

Reply:
It is not company name! it is just a internal name. The following fields is what users will see in the UI:

```
displayName: "redhat stroage operators"
  publisher: "Red Hat Storage"
```

Reply:
ack